algebird-core/src/main/scala/com/twitter/algebird/QTree.scala

```diff
@@ -367,3 +367,9 @@ case class QTree[A](
     (n.toDouble(ll) / luc, n.toDouble(uu) / ulc)
   }
 }
+
+case class QTreeAggregator(percentile: Double, k: Int) extends Aggregator[Double, QTree[Double], (Double, Double)] {
```

Comment:
How about making this generic for any `N:Numeric` instead of requiring `Double`?